robot-db/src/main/java/domain/Category.java

```diff
@@ -0,0 +1,33 @@
+package domain;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.Collection;
+
+/**
+ * Category Representation in Database
+ */
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class Category {
+
+    @Id
+    @GeneratedValue
+    @Getter
+    @Setter
+    private long categoryId;
+
+    @Getter
+    @Setter
+    @Enumerated(EnumType.STRING)
```

Comment:
I hope you DO have a proper note on your enumerated type that mentions that sorting and reordering is absolutely forbidden. :imp: